VagrantProvision.sh

```diff
@@ -19,6 +19,24 @@ sudo service ntp stop
 sudo ntpd -gq
 sudo service ntp start
 
+echo "### Installing Java 8..."
+sudo wget --quiet --no-check-certificate --no-cookies --header "Cookie: oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/8u112-b15/jdk-8u112-linux-x64.tar.gz -P /tmp
+sudo tar -xvzf /tmp/jdk-8u112-linux-x64.tar.gz --directory=/tmp >/dev/null
```

Comment:
Would like to see this wrapped in an `if` so provision doesn't always download this file if it's already present.